CarRentalCompany.Integration/Builders/CarReceiptFormBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using CarRentalCompany.Domain.Models;
+
+namespace CarRentalCompany.Integration.Builders
+{
+    public class CarReceiptFormBuilder
```

Comment:
Ten builder jest źle zbudowany. On jest wrapperem na model domenowy CarReceiptForm, a nie builderem. Albo jest to zbędna abstrakcja albo coś jest nie tak zbudowane w systemie